fileglancer/filestore.py

```diff
@@ -120,7 +120,8 @@ def get_file_info(self, path: Optional[str] = None) -> FileInfo:
         """
         full_path = self._check_path_in_root(path)
         stat_result = os.stat(full_path)
-        return FileInfo.from_stat(path, full_path,stat_result)
+        rel_path = os.path.relpath(full_path, self.root_path)
+        return FileInfo.from_stat(rel_path, full_path, stat_result)
```

Comment:
This change was mostly for type check

Reply:
Looks like this converts None and empty string to a `.`, which is arguably more correct. I've added a comment to make that purpose clear in the code. Please let me know if I've misunderstood.